testing/src/test/java/io/stargate/it/cql/JavaDriverTestBase.java

```diff
@@ -0,0 +1,99 @@
+package io.stargate.it.cql;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.Version;
+import com.datastax.oss.driver.api.core.config.DriverConfigLoader;
+import com.datastax.oss.driver.api.core.config.OptionsMap;
+import com.datastax.oss.driver.api.core.config.TypedDriverOption;
+import com.datastax.oss.driver.api.core.session.SessionBuilder;
+import com.datastax.oss.driver.internal.core.loadbalancing.DcInferringLoadBalancingPolicy;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.storage.ClusterConnectionInfo;
+import java.net.InetSocketAddress;
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+import net.jcip.annotations.NotThreadSafe;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+/**
+ * Base class for tests that use a dedicated Java driver session (and keyspace) for each method.
+ *
+ * <p>TODO maybe refactor this into a Junit extension
+ */
+@NotThreadSafe
+public abstract class JavaDriverTestBase extends BaseOsgiIntegrationTest {
```

Comment:
I think it would be nice to convert `BaseOsgiIntegrationTest` into a JUnit 5. WDYT?